test/__snapshots__/ember.test.js.snap

```diff
@@ -0,0 +1,303 @@
+    url.searchParams.set('url', this.currentURL);
+
+    // @ts-expect-error @rehearsal TODO TS2339: Property 'text' does not exist on type 'Readonly<EmptyObject>'.
+    if (this.args.text) {
+      // @ts-expect-error @rehearsal TODO TS2339: Property 'text' does not exist on type 'Readonly<EmptyObject>'.
+      url.searchParams.set('text', this.args.text);
+    }
+
+    // @ts-expect-error @rehearsal TODO TS2339: Property 'hashtags' does not exist on type 'Readonly<EmptyObject>'.
+    if (this.args.hashtags) {
+      // @ts-expect-error @rehearsal TODO TS2339: Property 'hashtags' does not exist on type 'Readonly<EmptyObject>'.
+      url.searchParams.set('hashtags', this.args.hashtags);
+    }
+
+    // @ts-expect-error @rehearsal TODO TS2339: Property 'via' does not exist on type 'Readonly<EmptyObject>'.
+    if (this.args.via) {
+      // @ts-expect-error @rehearsal TODO TS2339: Property 'via' does not exist on type 'Readonly<EmptyObject>'.
+      url.searchParams.set('via', this.args.via);
+    }
+
+    url.searchParams.set('locale', this.locale.current());
+
+    return url;
+  }
+}
+"
+`;
+
+exports[`validation-test ember@4.4 LTS > migrate 3`] = `
+"import Model, { attr } from '@ember-data/model';
+
+const COMMUNITY_CATEGORIES = ['Condo', 'Townhouse', 'Apartment'];
+
+export default class RentalModel extends Model {
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'title' implicitly has an 'any' type.
+  @attr title;
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'owner' implicitly has an 'any' type.
+  @attr owner;
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'city' implicitly has an 'any' type.
+  @attr city;
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'location' implicitly has an 'any' type.
+  @attr location;
+  // @ts-expect-error @rehearsal TODO TS2564: Property 'category' has no initializer and is not definitely assigned in the constructor.
+  @attr category: string;
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'image' implicitly has an 'any' type.
+  @attr image;
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'bedrooms' implicitly has an 'any' type.
+  @attr bedrooms;
+  // @ts-expect-error @rehearsal TODO TS7008: Member 'description' implicitly has an 'any' type.
+  @attr description;
+
+  get type() {
+    if (COMMUNITY_CATEGORIES.includes(this.category)) {
+      return 'Community';
+    } else {
+      return 'Standalone';
+    }
+  }
+}
+"
+`;
+
+exports[`validation-test ember@4.4 LTS > migrate 4`] = `
+"import Component from '@glimmer/component';
+
+export default class Hello extends Component {
+  name = 'world';
+
+  <template>
+    {{! @glint-expect-error @rehearsal TODO TS2339: Property 'age' does not exist on type '{}'. }}
```

Comment:
wouldn't we expect `age` is coming from the backing .ts file for this component?

Reply:
There are currently no code-fixes implemented in glint to stub the signature and put in inferred types.